### ISSUE TITLE
Error while running estimateDispersions()

### ISSUE THREAD

**Issue by kathirij** [cole-trapnell-lab/monocle-release#36]
Hello,
   I have an error when I estimate dispersion on my CDS. The data that I am feeding into it is UMI based (10X). The error that I get is: 

`gbm <- estimateDispersion(gbm)`
`Removing 140 outliers
Warning message:
Deprecated, use tibble::rownames_to_column() instead. `

I suspect that I have zeros or some other value in one of my cells. However, I am not sure how to find that or remove those. I am unable to attach the CDS file here - any suggestions about how to send it over?

**Comment by kathirij** [comment]
You're right. I meant a warning. However, my guess of some missing data is because of the downstream errors I get. For example: this is the error I get when I am trying to look at the mRNA distribution. 

`> pData(gbm)$Total_mRNAs <- Matrix:colSums(exprs(gbm))`
`Error in base::":"(from, to) : NA/NaN argument`
`In addition: Warning message:`
`In base::":"(from, to) :`
`  numerical expression has 3120 elements: only the first used`

**Comment by Payne-Wang** [comment]
> You're right. I meant a warning. However, my guess of some missing data is because of the downstream errors I get. For example: this is the error I get when I am trying to look at the mRNA distribution.
> 
> `> pData(gbm)$Total_mRNAs <- Matrix:colSums(exprs(gbm))`
> `Error in base::":"(from, to) : NA/NaN argument`
> `In addition: Warning message:`
> `In base::":"(from, to) :`
> ` numerical expression has 3120 elements: only the first used`

hi. I also met this warning and I found the output of the data in some samples included NaN after treated by the function relative2abs to convert TPM to relative mRNA count. Have you already solve this question? How to deal with it?
Thank you!
